.github/workflows/deploy.yml

```diff
@@ -0,0 +1,58 @@
+name: Deployment
+
+on:
+  # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch
+  workflow_dispatch:
+    branch:
+      description: 'The branch to deploy'
+      default: 'main'
+      required: false
+      type: string
+
+env:
+  NODE_ENV: development
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Is the GITHUB_TOKEN used?

Reply:
At the moment, no we don't need it but we will if we want to trigger a GH deploy dispatch event